src/plugins/CMakeLists.txt

```diff
@@ -120,6 +120,7 @@ add_subdirectory(key_publisher)
 add_subdirectory(publisher)
 add_subdirectory(scene3d)
 add_subdirectory(screenshot)
+add_subdirectory(shutdown_button)
```

Comment:
Alphabetize

Reply:
I don't understand... `screenshot < shutdown < teleop` ... or is it too late already?